server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/CloudFoundryApplication.java

```diff
@@ -54,6 +66,14 @@ public void addEnvironmentVariables(String environmentVariableName) {
         return services;
     }
 
+    public ArrayList<CloudFoundryService> getServicesMatchedToProvider() {
+        return servicesMatchedToProvider;
```

Comment:
Do you need write access on this list? if not: Consider making it read only using  [`Collections.unmodifiableList()`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#unmodifiableList-java.util.List-)